istio-operator/istio-operator.spec

```diff
@@ -13,15 +13,26 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit 510e2d4075b2284e0c2820988b96576503a9897c
+%global git_commit dda6d0aaada2d5bdaf82a6f1ce316877673c829a
 %global git_shortcommit  %(c=%{git_commit}; echo ${c:0:7})
 
 %global provider        github
 %global provider_tld    com
 %global project         maistra
 %global repo            istio-operator
+
+# charts
+%global charts_git_commit 4408997686d86a157f006403acfd46508d166f8c
+%global chargs_git_shortcommit  %(c=%{charts_git_commit}; echo ${c:0:7})
+
+%global charts_repo      istio
+%global charts_version   1.1.0
```

Comment:
1.1.1

Reply:
this path is hard coded in the operator itself.  can we leave it as 1.1.0 for now and revisit in a following sprint?  we need a better design than what we have.